thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/alert/content/EmailContentFormatterContext.java

```diff
@@ -1,12 +1,32 @@
 package com.linkedin.thirdeye.alert.content;
 
+import com.linkedin.thirdeye.datalayer.dto.AlertConfigDTO;
+import com.linkedin.thirdeye.datalayer.dto.AnomalyFunctionDTO;
 import org.joda.time.DateTime;
 
 
 public class EmailContentFormatterContext {
+  private AnomalyFunctionDTO anomalyFunctionSpec;
+  private AlertConfigDTO alertConfig;
```

Comment:
How is this related to the dummy-function workaround?

Reply:
This is not related to the workaround. It is to fix when there is no anomaly generated, we might face null pointer exception when generating emails.